local/snapshot.go

```diff
@@ -178,8 +180,10 @@ func (ln *localNetwork) loadSnapshot(
 ) error {
 	ln.lock.Lock()
 	defer ln.lock.Unlock()
+	// TODO for CLI team -- what are the lines below supposed to do?
+	// The linter is complaining about them.
 	snapshotDir := filepath.Join(ln.snapshotsDir, snapshotPrefix+snapshotName)
-	snapshotDbDir := filepath.Join(filepath.Join(snapshotDir, defaultDbSubdir))
+	snapshotDbDir := filepath.Join(filepath.Join(snapshotDir, defaultDBSubdir))
```

Comment:
@felipemadero same here, the first `Join` is not doing anything.

Reply:
same